plugins/premium/mentions.md

```diff
@@ -291,7 +292,7 @@ tinymce.init({
 
 {{site.requires_5_6v}}
 
-If `mentions_select` is resolved with an object specifying type and user details, a predefined select template will be used. The only valid template at this time is `'profile'`. For details on the user properties required for the `profile` template, see: [User properties](#userproperties).
+If `mentions_select` is resolved with an object specifying the type and user details, a predefined select template will be used. To use the predefined template, set `type` to `'profile'`. See [`Example: Using mentions_select`](#exampleusingmentions_select) for an example of how this is done. For details on the user properties required for the `profile` template, see: [User properties](#userproperties).
```

Comment:
```suggestion
If `mentions_select` is resolved with an object specifying the type and user details, a predefined select template will be used. To use the predefined template, set `type` to `'profile'`. For details on the user properties required for the `profile` template, see: [User properties](#userproperties).
```
That example doesn't really show how to set the option to a predefined template, it just mentions that you can, same as here.

Reply:
Also, "predefined select template"? Should it just be "predefined template"?

Reply:
Lee clarified what "select template" means, but it's not obvious to me. Would "hover template" or "popup template" be better?